src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -209,20 +212,20 @@ private IssuesSnapshot TranslateMarkerSpans()
 
         private void SnapToNewSnapshot(IssuesSnapshot newIssues)
         {
+            var oldIssues = Factory.CurrentSnapshot;
+
             // Tell our factory to snap to a new snapshot.
             this.Factory.UpdateSnapshot(newIssues);
 
-            Provider.RefreshErrorList();
+            sonarErrorDataSource.RefreshErrorList();
 
             // Work out which part of the document has been affected by the changes, and tell
             // the taggers about the changes
-            SnapshotSpan? affectedSpan = CalculateAffectedSpan(LastIssues, newIssues);
+            SnapshotSpan? affectedSpan = CalculateAffectedSpan(oldIssues, newIssues);
             foreach (var tagger in activeTaggers)
             {
                 tagger.UpdateMarkers(newIssues.IssueMarkers, affectedSpan);
             }
-
-            this.LastIssues = newIssues;
```

Comment:
This is the only remaining place `LastIssues` was being used so there's no longer any need for it to be stored - we can just fetch the old snapshot in this method before updating to the new one.